lib/Routes/reports.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+const dataService = require('../../models/data-service');
+const tableName = 'reports';
+const DebugDB = require('../../debuggers').server;
```

Comment:
I think this should be `debugDB`? Otherwise it implies this is a constructor.
```suggestion
const debugDB = require('../../debuggers').server;
```